src/qtt/measurements/videomode.py

```diff
@@ -465,6 +469,17 @@ def run(self, start_readout=True):
         if start_readout:
             if self.verbose:
                 print('%s: run: startreadout' % (self.__class__.__name__,))
+            if isinstance(self.minstrumenthandle, AcquisitionScopeInterface):
+                if scan_dimensions == 1:
+                    self.minstrumenthandle.period = 1e-3
```

Comment:
the abc class `AcquisitionScopeInterface` does not have setters for `period`, `number_of_samples` and `enabled_channels`